File: ECommerceApp/DataModel/Taxes+CoreDataProperties.m
//
//  Taxes+CoreDataProperties.m
//  ECommerceApp
//
//  Created by Prateek Raj on 11/07/18.
//  Copyright © 2018 Prateek Raj. All rights reserved.
//
//

#import "Taxes+CoreDataProperties.h"

@implementation Taxes (CoreDataProperties)

+ (NSFetchRequest<Taxes *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Taxes"];
}

@dynamic value;
@dynamic name;
@dynamic products;

@end

File: ECommerceApp/DataModel/Taxes+CoreDataProperties.swift
//
//  Taxes+CoreDataProperties.swift
//  ECommerceApp
//
//  Created by Prateek Raj on 28/07/18.
//  Copyright © 2018 Prateek Raj. All rights reserved.
//
//

import Foundation
import CoreData


extension Taxes {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Taxes> {
        return NSFetchRequest<Taxes>(entityName: "Taxes")
    }

    @NSManaged public var name: String?
    @NSManaged public var value: Float
    @NSManaged public var products: Product?

}
